code/Attack.ipynb

```diff
@@ -0,0 +1,1466 @@
+{
```

Comment:
<p>Can you provide a visual example of an example generated using this method would look like? And also provide the reference that introduced this attack. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/robustness-foundation-models/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>